test/recipes/30-test_evp_data/evppkey_rsa_common.txt

```diff
@@ -919,6 +919,7 @@ Input="0123456789ABCDEF0123456789ABCDEF"
 Output=4DE433D5844043EF08D354DA03CB29068780D52706D7D1E4D50EFB7D58C9D547D83A747DDD0635A96B28F854E50145518482CB49E963054621B53C60C498D07C16E9C2789C893CF38D4D86900DE71BDE463BD2761D1271E358C7480A1AC0BAB930DDF39602AD1BC165B5D7436B516B7A7858E8EB7AB1C420EEB482F4D207F0E462B1724959320A084E13848D11D10FB593E66BF680BF6D3F345FC3E9C3DE60ABBAC37E1C6EC80A268C8D9FC49626C679097AA690BC1AA662B95EB8DB70390861AA0898229F9349B4B5FDD030D4928C47084708A933144BE23BD3C6E661B85B2C0EF9ED36D498D5B7320E8194D363D4AD478C059BAE804181965E0B81B663158A
 
 # Verify using salt length auto detect
+FIPSversion = <3.4.0
```

Comment:
Why should this fail?

Reply:
The recovered saltlen is 30, which is larger than the output block length (20) of SHA-1.

Reply:
So the signature data is actually invalid? (This saltlen rule is not new - it was in 186-4 also)

Reply:
I would say that the signature data is valid but not FIPS-compliant because `0 <= sLen <= hLen` is an additional constraint defined in FIPS 186.